test-utils/src/lib.rs

```diff
@@ -122,7 +122,7 @@ macro_rules! impl_frame_system {
 
 pub type Moment = u64;
 parameter_types! {
-	pub const MinimumPeriod: Moment = 1;
+	pub const MinimumPeriod: Moment = BLOCKTIME / 2;
```

Comment:
Changed this while trying another approach, but I thought why not use realistic numbers.